stdlib/websub/src/main/ballerina/src/websub/hub_service.bal

```diff
@@ -401,11 +401,11 @@ function persistTopicRegistrationChange(string mode, string topic) returns error
     }
 }
 
-# Function to add/change/remove the persisted subscription details.
+# Adds/Changes/Removes the persisted subscription details.
 #
-# + mode - Whether the subscription change is for unsubscription/unsubscription
+# + mode - Whether the subscription change is for subscription/unsubscription
 # + subscriptionDetails - The details of the subscription changing
-# + return - `error` if an error occurred while persisting the change, `()` otherwise
+# + return - An `error` if an error occurred while persisting the change or else `()`
```

Comment:
```suggestion
# + return - An `error` if an error occurred while persisting the change or else `()`
```
Need to specify the error.

Reply:
We are using a common error in this module. So we don't need to specify the error.